### PR TITLE
Require Python 3.6 (f-strings) and fix pylints

### PR DESCRIPTION
**Pull request by def-**
So far only done scripts directory, will do the rest if this is
considered good

## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [x] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by C0D3D3V** [comment]
how a bout using black for formatting python scripts?

**Comment by def-** [comment]
Yes, good idea. As long as it keeps the diff to our current state relatively small.

File: scripts/check_identifiers.py [hunk 1]
```diff
@@ -65,7 +65,7 @@ def main():
 		error = check_name(i["kind"], i["qualifiers"], i["type"], i["name"])
 		if error:
 			unclean = True
-			print("{}:{}:{}: {}: {}".format(i["file"], i["line"], i["column"], i["name"], error))
+			print(f"{['file']}:{i['line']}:{i['column']}: {i['name']}: {error}")
```
Comment (heinrich5991) [inline scripts/check_identifiers.py:68]:
```suggestion
			print(f"{i['file']}:{i['line']}:{i['column']}: {i['name']}: {error}")
```

**Comment by C0D3D3V** [comment]
Btw. there is a tool called [flynt](https://github.com/ikamensh/flynt) that can convert old format strings into f strings automaticly. It works pretty well, on to complex formats or format that would result in an different output (depending on the input type), it does not change the format. 

I have used it multiple times on some projects. 
This could be combined with adding a formatter to our project.

**Comment by def-** [comment]
Ready for review.

**Comment by C0D3D3V** [comment]
I like it, looks a lot more readable :)
Next step would be to add black as formatter.

**Comment by heinrich5991** [comment]
bors r+

Thanks.

**Pull request merged**